### PR TITLE
Improce expression for chapter_preliminaries/linear-algebra.md

### PR DESCRIPTION
**Pull request by PeterH0323**
Improce expression for `chapter_preliminaries/linear-algebra.md`

### CODE REVIEW

File: chapter_preliminaries/linear-algebra.md [hunk 1]
```diff
@@ -474,17 +475,17 @@ A_sum_axis1, A_sum_axis1.shape
 沿着行和列对矩阵求和，等价于对矩阵的所有元素进行求和。
 
 ```{.python .input}
-A.sum(axis=[0, 1])  # SameasA.sum()
+A.sum(axis=[0, 1])  # 结果和 A.sum() 相同
```
Comment (astonzhang) [inline chapter_preliminaries/linear-algebra.md:478]:
```suggestion
A.sum(axis=[0, 1])  # 结果和A.sum()相同
```

File: chapter_preliminaries/linear-algebra.md [hunk 2]
```diff
@@ -474,17 +475,17 @@ A_sum_axis1, A_sum_axis1.shape
...
 ```
 
 ```{.python .input}
 #@tab pytorch
-A.sum(axis=[0, 1])  # SameasA.sum()
+A.sum(axis=[0, 1])  # 结果和 A.sum() 相同
```
Comment (astonzhang) [inline chapter_preliminaries/linear-algebra.md:483]:
```suggestion
A.sum(axis=[0, 1])  # 结果和A.sum()相同
```

File: chapter_preliminaries/linear-algebra.md [hunk 3]
```diff
@@ -474,17 +475,17 @@ A_sum_axis1, A_sum_axis1.shape
...
 ```
 
 ```{.python .input}
 #@tab tensorflow
-tf.reduce_sum(A, axis=[0, 1])  # Sameastf.reduce_sum(A)
+tf.reduce_sum(A, axis=[0, 1])  # 结果和 tf.reduce_sum(A) 相同
```
Comment (astonzhang) [inline chapter_preliminaries/linear-algebra.md:488]:
```suggestion
tf.reduce_sum(A, axis=[0, 1])  # 结果和tf.reduce_sum(A)相同
```

File: chapter_preliminaries/linear-algebra.md [hunk 4]
```diff
@@ -7,8 +7,9 @@
 
 ## 标量
 
+
 如果你曾经在餐厅支付餐费，那么应该已经知道一些基本的线性代数，比如在数字间相加或相乘。
-例如，北京的温度为$52^{\circ}F$（除了摄氏度外，另一种温度计量单位）。
+例如，北京的温度为$52^{\circ}F$（华氏度，约11.1°C）。
```
Comment (astonzhang) [inline chapter_preliminaries/linear-algebra.md:12]:
```suggestion
例如，北京的温度为$52^{\circ}F$（华氏度，除摄氏度外的另一种温度计量单位）。
```
